api/helpers.go

```diff
@@ -407,6 +407,18 @@ func normalizeProperty(unitName string, p map[string]interface{}, d DCOSHelper)
 		unitOutput += fmt.Sprintf("%s state is not one of the possible states %s. Current state is [ %s ]. Please check `systemctl show all %s` to check current unit state. ", unitName, okStates, p["ActiveState"], unitName)
 	}
 
+	// check if the unit is flapping
+	if p["ActiveState"].(string) == "activating" {
```

Comment:
that's what we got from dbus lib
https://github.com/coreos/go-systemd/blob/master/dbus/methods.go#L176